### ISSUE TITLE
[FEATURE] render keyboard tags

### ISSUE THREAD

**Issue by Su24nn3** [Zettlr/Zettlr#4398]
### Checklist

- [X] Zettlr does not possess this feature (if it does, but the feature doesn't work out for you, propose an enhancement instead.)
- [X] Zettlr's current features cannot be used to approximate this behaviour (otherwise, propose an enhancement instead.)
- [X] I have searched the documentation to make sure I did not overlook it.
- [X] I have searched both open and closed issues to see if someone has already proposed this.

### Feature Proposal

With other markdown editors/previewers, I use the `<kbd>` / `<key>` tag a lot, and on Github too : <kbd>Ctrl</kbd>
This is not really basic markdown syntax, but I think it is very useful to display keyboard shortcuts for example.

An easy and readable workaround is to use code syntax instead, but I find it ambiguous and less readable than having both.

What do you think about adding this feature to Zettlr ?

### Example Workflow

Not sure how to implement this to follow WYSIWYM Zettlr philosophy. 
Ideally, I think the tag itself (the `<kbd>` part) should not be displayed, only its content, or be displayed only when text is selected, then it could be in a smaller font and greyed, and you could quickly convert selected text with a button, like when selecting a text and making it bold/italic/link/etc. (sorry, I couldn't find how you call this little menu that appears when text is selected).

### Caveats

I don't know, I guess the tag syntax is a little bulky ?

### Do you wish to attempt implementing this yourself?

No

### Platforms

- [X] Windows
- [X] macOS
- [X] Linux

### Additional Information

Unfortunately I don't think I have the programming skills to attempt to implement this, but I thought I would suggest it anyway, if anyone is interested.
Thank you for this amazing app, I think it distinguishes itself a lot from other markdown previewers, text editor and other note-taking apps out there !

**Comment by nathanlesage** [comment]
Yeah, I can see the value in this. I don't have too much time on my hand, but the rendering of kbd-tags could be done using the emphasis renderer since it's strictly speaking an inline tag. But then, I have to fix that renderer at some point as well … so don't expect this feature too early, but I think it makes sense.
